src/arrays/arrays_with_methods/hard/q12/ArrayOfFacts.java

```diff
@@ -0,0 +1,42 @@
+package arrays.arrays_with_methods.hard.q12;
+
+/*
+Write a Java program that asks the user to input the length of an array called n. Then, the program initializes an
+integer array with a series of factorial numbers 1!, 2!, 3!, 4!, ...etc. up to n! And prints the array. Recall that n!
+is calculated as follows:
+n! = n x (n - 1) x (n - 2) x … x 1.
+In particular, write a method called factorial that takes an integer and returns the factorial of that number (e.g.,
+calling factorial(3) returns 6).
+Sample output:
+Enter the size of the array:
+5
+1
+2
+6
+24
+120
+ */
+
+import java.util.Scanner;
+
+public class ArrayOfFacts {
+    public static void main(String[] args) {
+        Scanner input = new Scanner(System.in);
+        System.out.println("Enter the size of the array: ");
+        int n = input.nextInt();
+        int[] arr = new int[n];
```

Comment:
I don't think we can actually make sure that they use an array here because they can just get the factorial and print it out. We should probably change this question.